js/react_native/ios/Podfile

```diff
@@ -1,12 +1,16 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, '13.0'
+platform :ios, '15.0'
```

Comment:
Why is this change needed?

Reply:
Specs satisfying the `RCTRequired (from `../node_modules/react-native//Libraries/Required`)` dependency were found, but they required a higher minimum deployment target.
https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=1439006&view=logs&j=de302ec2-2305-57e0-e8c6-cd89c569f2a3&t=65d5dc62-c4eb-539e-a5bd-5c7416a33c7a&l=63